packages/app/src/App.tsx

```diff
@@ -161,7 +161,10 @@ const routes = (
     <Route path="/home" element={<HomepageCompositionRoot />}>
       {homePage}
     </Route>
-    <Route path="/catalog" element={<CatalogIndexPage />} />
+    <Route
+      path="/catalog"
+      element={<CatalogIndexPage initiallySelectedNamespaces={['default']} />}
```

Comment:
perhaps we can leave this as it was and remove `different-namespace.yaml`?

Reply:
Yes, this was not meant to be checked in, was part of my manual testing. I would like to keep `different-namespace.yaml` as it's handy to have an example in a different namespace as the current ones are all in the default

Reply:
These changes have been made, I did remove `different-namespace.yaml` as well, it's easy to recreate if needed 👍